bindings/cs/MongoDB.Libmongocrypt/KmsType.cs

```diff
@@ -1,27 +0,0 @@
-/*
- * Copyright 2019–present MongoDB, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-namespace MongoDB.Libmongocrypt
-{
-    /// <summary>
-    /// Type of Kms
-    /// </summary>
-    public enum KmsType
```

Comment:
I've removed this class. Now, if we need to add a new kms provider (without some specific cases). We will just need to update the version of `mongocrypt.dll`. No bindings changes are required.